docs/api.rst

```diff
@@ -0,0 +1,232 @@
+.. _api:
+
+API
+===
+
+As mentioned in the :ref:`design` section, Babble communicates with the App 
+through an ``AppProxy`` interface, which exposes three methods for Babble to 
```

Comment:
Here you should only mention the BabbleProxy, the details of the AppProxy are reserved to the babble implementation